src/Microsoft.AspNetCore.Mvc.Core/Controllers/IControllerFactory.cs

```diff
@@ -18,6 +18,7 @@ public interface IControllerFactory
         /// <summary>
         /// Releases a controller instance.
         /// </summary>
+        /// <param name="context"><see cref="ControllerContext"/> for the action to execute.</param>
```

Comment:
`the executing action` is better since this is a `Release()` method, not about to execute.